.github/workflows/reusable-testing.yml

```diff
@@ -3,11 +3,16 @@ name: Testing
 on:
   workflow_call:
     inputs:
-      minimum-php:
-        description: 'Minimum PHP version to test against.'
+      matrix-php-versions-unit:
+        description: 'JSON string array of PHP versions to use in unit test matrix.'
         type: string
         required: false
-        default: '5.6'
+        default: "['5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3']"
+      matrix-php-versions-functional:
+        description: 'JSON string array of PHP versions to use in functional test matrix.'
+        type: string
+        required: false
+        default: "['7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3']"
```

Comment:
Out of curiosity, what happens when this is invalid JSON?

Reply:
With value `"'7.1', '7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3']"` (I just deleted the leading bracket)
> Error when evaluating 'strategy' for job 'unit'. brianhenryie/bh-wp-github-actions-tests/.github/workflows/reusable-testing.yml@master (Line: 26, Col: 14): Unexpected value '7.1'